### DISCUSSION TITLE
Width of Left_drawer

### DISCUSSION THREAD

**Discussion by TimBrrll** [zauberzeug/nicegui#840] [Q&A] [answered]
### Question

Hello guys, 

I have the question on how to change the width of the ui.left_drawer element, as its width is predefined to be 300px. I have tried .style, .classes or .props, however its change is too highlevel once you look into the source code of the created webpage. Do you have any ideas on how to solve this?

**Comment by tjakobi** [comment, marked as answer]
This is possible using `props()`, see https://github.com/zauberzeug/nicegui/discussions/161#discussioncomment-4153697:

```python
@ui.page('/')
def page():
    ui.label('Content')
    with ui.left_drawer().props('width=80 bordered'):
        ui.label('Drawer')
```
